syntax = "proto3";
package c15_grpc;
service Greeter {
  rpc SayHello (HelloRequest) returns (HelloReply) {}
  rpc CalculateArea (GiveSide) returns (AreaReply) {}
}

message HelloRequest {
  string  name = 1;
}

message HelloReply {
  string response = 1;
}

message  GiveSide {
  int32 side1 = 1;
  int32 side2 = 2;
}

message AreaReply {
  int32 area = 1;
}